### PR TITLE
Cross-build for JS+Native

### PR DESCRIPTION
**Pull request by armanbilge**
Almost there, just have to track down a couple things ... :)

### CODE REVIEW

**Comment by armanbilge** [comment]
Darn.
- https://github.com/typelevel/skunk/pull/890

File: modules/core/src/test/scala/GraphQLResponseTests.scala [hunk 1]
```diff
@@ -9,16 +9,16 @@ import munit.Assertions._
 
 object GraphQLResponseTests {
   def assertWeaklyEqual(x: Json, y: Json, strictPaths: List[List[String]] = Nil): Unit =
-    assert(weaklyEqual(x, y, strictPaths))
+    assert(weaklyEqual(clue(x), clue(y), strictPaths))
```
Comment (armanbilge) [inline modules/core/src/test/scala/GraphQLResponseTests.scala:12]: Anything wrapped in `clue` will be printed when the assertion fails, useful for debugging.

File: modules/sql/shared/src/test/scala/SqlWorldSuite.scala [hunk 2]
```diff
@@ -1559,6 +1559,9 @@ trait SqlWorldSuite extends CatsEffectSuite {
       assertIO(resTotal, Some(expected))
   }
 
+  // JS treats floats as doubles
+  def isJS = Option(System.getProperty("java.vm.name")).contains("Scala.js")
```
Comment (armanbilge) [inline modules/sql/shared/src/test/scala/SqlWorldSuite.scala:1563]: This was highly annoying but does the trick ...

File: modules/sql/js-jvm/src/test/scala/SqlDatabaseSuite.scala [hunk 3]
```diff
@@ -26,7 +25,7 @@ trait SqlDatabaseSuite extends CatsEffectSuite {
   }
 
   def bindPath(path: String): String =
-    Path(".").absolute.parent.flatMap(_.parent).flatMap(_.parent).get.toString+"/"+path
+    buildinfo.BuildInfo.baseDirectory + "/" + path
```
Comment (milessabin) [inline modules/sql/js-jvm/src/test/scala/SqlDatabaseSuite.scala:28]: This is much better ... thanks!

File: modules/sql/shared/src/test/scala/SqlWorldSuite.scala (hunk 2 above)
Comment (milessabin) [reply modules/sql/shared/src/test/scala/SqlWorldSuite.scala:1563]: Ouch, but :+1:

**Review by milessabin — APPROVED** [review]
This is awesome! Thanks so much :tada:

**Pull request merged**